plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/UndefinedServletElement.java

```diff
@@ -26,15 +26,16 @@
   private static final String markerId = 
       "com.google.cloud.tools.eclipse.appengine.validation.undefinedServletMarker";
   private static final int severity = IMarker.SEVERITY_ERROR;
-  private String servletClass;
+  private String servletClassName;
   
-  public UndefinedServletElement(String servletClass, DocumentLocation start, int length) {
-    super(Messages.getString("undefined.servlet.class"), markerId, severity, start, length);
-    this.servletClass = servletClass;
+  public UndefinedServletElement(String servletClassName, DocumentLocation start, int length) {
+    super(servletClassName + " " + Messages.getString("undefined.servlet.class"),
```

Comment:
For I18N purposes this should be done with a message format rather than string concatenation  in case the class name is not the first word in some non-English language. E.g.

Messages.getString("%s undefined.servlet.class", servletClassName)